README.md

```diff
@@ -420,3 +420,16 @@ global.$ = global.jQuery = $;
 The same declaration can be applied to other vendor libraries.
 
 Reference: https://github.com/facebook/jest/issues/708
+
+### Failing with Node < v6
+
+By default we use JSDOM v13, which requires Node v6. If you want to use Node in a lower version than 6 to run your tests, make sure to configure jest to use an older JSDOM version, either in your `jest.config.js`, `jest.config.json`, `package.json`:
```

Comment:
Since v12 jsdom requires Node 8+: https://github.com/jsdom/jsdom/blob/master/Changelog.md#1200

Reply:
You are right, I got confused!